core/src/main/java/nu/yona/server/messaging/service/BuddyMessageEmbeddedUserDTO.java

```diff
@@ -0,0 +1,40 @@
+package nu.yona.server.messaging.service;
+
+import java.util.Collections;
+import java.util.Date;
+import java.util.Map;
+import java.util.UUID;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import nu.yona.server.subscriptions.service.UserDTO;
+
+public abstract class BuddyMessageEmbeddedUserDTO extends BuddyMessageDTO
+{
+	private Map<String, Object> embeddedResources = Collections.emptyMap();
+
+	protected BuddyMessageEmbeddedUserDTO(UUID id, Date creationTime, UserDTO user, String nickname, String message)
+	{
+		super(id, creationTime, user, nickname, message);
+	}
+
+	protected BuddyMessageEmbeddedUserDTO(UUID id, Date creationTime, UUID targetGoalConflictMessageOriginID, UserDTO user,
+			String nickname, String message)
+	{
+		super(id, creationTime, user, nickname, message);
+	}
+
+	@JsonProperty("_embedded")
```

Comment:
This is not really new. On ``UserResource``, we do the same, but we cannot do that for polymorphic types. The resource assembler is responsible for filling the embedded map, but cannot define it.